scss/bitstyles/utilities/gap/_settings.scss

```diff
@@ -3,7 +3,7 @@
 $sizes: (
   'xs': size.get('xs'),
   's': size.get('s'),
-  'm': size.get('l'),
-  'l': size.get('xl'),
+  'm': size.get('m'),
+  'l': size.get('l'),
```

Comment:
This means we lost the `xl` -sized gap, and gained the `m`-sized gap. Probably we can:

- drop `m`, since it’s not been used anywhere or considered useful for the default config so far (unless you think it should be?)
- add `xl`
- rename places `u-gap-l` was used to `u-gap-xl`
- rename places `u-gap-m` ws used to `u-gap-l`

Then we end up with the same grid gaps, but with the names now matching the names of sizes in the design tokens map, wdyt?